src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -337,11 +337,11 @@ internal static MethodSymbol FindContainingGenericMethod(Symbol symbol)
             return null;
         }
 
-        private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken argPassingKeyword, out SyntaxToken paramsKeyword, out SyntaxToken thisKeyword)
+        private static RefKind GetModifiers(SyntaxTokenList modifiers, out SyntaxToken refOrOutKeyword, out SyntaxToken paramsKeyword, out SyntaxToken thisKeyword)
```

Comment:
>refOrOutKeyword [](start = 87, length = 15)

Shouldn't we eliminate this argument and all of its usages, since now ref-ness is not determined by a single token?